extensions/2.0/Vendor/ADOBE_materials_clearcoat_tint/README.md

```diff
@@ -0,0 +1,79 @@
+
+```json
+{
+    "materials": [
+        {
+            "name": "varnish",
+            "extensions": {
+                "KHR_materials_clearcoat": {
+                    "clearcoatFactor": 1.0
+                },
+                "ADOBE_materials_clearcoat_tint": {
+                    "clearcoatTintFactor": [
+                      0.4,
+                      0.1,
+                      0.1
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+
+### Clearcoat Tint
+
+All implementations should use the same calculations for the BRDF inputs. Implementations of the BRDF itself can vary based on device performance and resource constraints. See [Appendix B](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#appendix-b-brdf-implementation) for more details on the BRDF calculations.
+
+As with the core glTF 2.0 spec, all coloured textures are assumed to be sRGB and all coloured factors are assumed to be linear.
+
+|                                  | Type                                                                            | Description                            | Required             |
+|----------------------------------|---------------------------------------------------------------------------------|----------------------------------------|----------------------|
+|**clearcoatTintFactor**    | `number[3]`                                                                                       | The clearcoat tint's colour factor.  | No, default: `[1.0, 1.0, 1.0]`   |
+|**clearcoatTintTexture**   | [`textureInfo`](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#reference-textureinfo)  | The clearcoat layer's tint texture.  | No                               |
+
+The clearcoat tint defines the transmittance of light through the coat. That is, it defines what colour of light is allowed to pass through. As with other factor/texture combinations, these two values are multiplied.
+```
+clearcoat_tint.rgb = clearcoatTintFactor.rgb * clearcoatTintTexture.rgb
+```
+The `clearcoat_tint` modifies the formula for `f_clearcoat` (as defined in [KHR_materials_clearcoat](/extensions/2.0/Khronos/KHR_materials_clearcoat/README.md)) only slightly. The `(f_emissive + f_diffuse + f_specular)` term is simply multiplied by the clearcoat tint to represent the absorption of all light coming from the layer below.
+
+```
+clearcoatBlendFactor = clearcoatTexture.r * clearcoatFactor
+clearcoatFresnel = fresnel(0.04, NdotV)
+
+color = mix(vec3(1.0), clearcoat_tint, clearcoatBlendFactor) * (f_emissive + f_diffuse + f_specular) * (1.0 - clearcoatBlendFactor * clearcoatFresnel) +
+        f_clearcoat * clearcoatBlendFactor
+```
+
+## Schema
+
+- [glTF.ADOBE_materials_clearcoat_tint.schema.json](schema/glTF.ADOBE_materials_clearcoat_tint.schema.json)
```

Comment:
Should this be aligned with #2313?

Reply:
It looks like those changes were only applied to KHR_ extensions, not vendor extensions.

Reply:
That PR indeed updated only KHR extensions. Most likely one of older KHR files was used as a template for this extension so the naming consistency changes are applicable here as well.

Reply:
Done